code/api/src/seeders/20200825050120-quiz.js

```diff
@@ -0,0 +1,350 @@
+"use strict";
```

Comment:
Unlike rails, seeding is just like migrating in JS. I cannot edit directly on an existing seed file (took rails for granted). The flow of seed migrations using sequelize (if it gives you an error but you know you have sequelize installed, just keep checking out of different directories and eventually it works)

1. `npx sequelize-cli seed:generate --name quiz`
2. generates this file and you fill it in with information you want seeded
3. `npx sequelize-cli db:seed:all`

**If you make a mistake(again if command does not work, change directories):**

Reverse Previous Migration `node_modules/.bin/sequelize db:seed:undo`
Reverse All Migrations `node_modules/.bin/sequelize db:seed:undo:all`

If you deleted a seed file and you get an error that it can't find the file, retrieve it from the trash or create files with the same name as the one missing. Then after you successfully pass undo:all, run step 3 again to create new migrations.

If you keep seeing this `nodemon app crashed - waiting for file changes before starting`, run the setup again. So in this case, checkout to code and run `npm run setup`

Reply:
Resource: https://sequelize.org/master/manual/migrations.html#running-seeds